src/env/node.js

```diff
@@ -39,13 +52,21 @@ function getVisibleGroups (node, options, groups = []) {
 
 function getTree (msg, i) {
 	if (msg.collapsed !== undefined) {
+		const icons = {
+			collapsed: "▷",
+			expanded: "▽",
+			highlightedCollapsed: "▶︎",
+			highlightedExpanded: "▼",
```

Comment:
Nit: I'd put the adjective after, e.g. `collapsed_highlighted`, since it's a variation of collapsed.

Reply:
Agreed! Thanks.